test.sh

```diff
@@ -1,7 +1,8 @@
 #! /bin/bash
```

Comment:
nit: 
```suggestion
#!/usr/bin/env bash
```

Reply:
Can you please explain this suggestion?

Reply:
It's a common pattern used in other scripts (see spell repos), I specified it as a nit as not related to the PR changes but just for consistency.
The rational behind the shift in pattern I guess is related to giving more flexibility on different systems.

Reply:
Yes, this increases portability.  I support this change.

Reply:
Addressed in #37